app/assets/stylesheets/modules/homepage.scss

```diff
@@ -0,0 +1,44 @@
+@import "variables";
+@import "mixins";
+
+body.home-page {
+  background: url(dog.png) no-repeat;
+  background-size: 100%;
+  opacity: .9;
+
+  .right {
+    padding: 1em;
+    padding-left: 2em;
+
+    a {
+      font-family: "Poiret One";
+      color: #fff;
+      font-size: 0.9em;
+      margin-right: 1.5em;
+      border-bottom: 2px solid transparent;
+      padding-bottom: 0.3em;
+
+      &:hover {
+        border-color: teal;
```

Comment:
Color `teal` should be written in hexadecimal form as `#008080`<br>Color literals like `teal` should only be used in variable declarations; they should be referred to via variable everywhere else.